### DISCUSSION TITLE
openapi.json for common schema

### DISCUSSION THREAD

**Discussion by goldingdamien** [encode/django-rest-framework#8620] [Question & Answer] [answered]
Hello, I am looking for an openapi file for default paths provided by django.

I am unfamiliar with django and have limited experience in python.
Instead of generating an openapi file, it would be useful for developers like me if there was an openapi file that could easily be 
integrated without having to research an unfamiliar language/framework.

If there is already an openapi file available for this, please could you share the link to it.
If there is not, I would like to request it.
If it can be generated, I would still like to request a static file, but would look into generating if someone could share how to do so.
If I am in someone mistaken in believing that django provides default paths for authentication, etc., and that instead a lot of it is dynamic, please let me know.

Regards.

**Comment by lovelydinosaur** [comment, marked as answer]
You probably want to take a look at this package... https://github.com/axnsan12/drf-yasg/
